net-mgmt/nrpe3/src/opnsense/mvc/app/models/OPNsense/Nrpe/Command.xml

```diff
@@ -0,0 +1,27 @@
+<model>
+    <mount>//OPNsense/nrpe/command</mount>
+    <description>NRPE command configuration</description>
+    <version>0.0.1</version>
+    <items>
+        <commands>
+            <command type="ArrayField">
+                <enabled type="BooleanField">
+                    <default>1</default>
+                    <Required>Y</Required>
+                </enabled>
+                <name type="TextField">
+                    <default></default>
+                    <Required>Y</Required>
+                    <mask>/^([0-9a-zA-Z_\-]){0,32}$/u</mask>
```

Comment:
```suggestion
                    <mask>/^[0-9a-z_\-]{0,32}$/ui</mask>
```

please give that one a try. BTW: Please consider to change either the regex or the required check - they do not fit together (the regex allows zero length, the required not.